packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -555,203 +755,148 @@ export default function ExperimentHeader({
+                    </DropdownMenuItem>
                   )}
-                {editPhases && !isBandit && (
-                  <button
-                    className="dropdown-item"
-                    onClick={() => editPhases()}
+                  <DropdownMenuItem onClick={() => setAuditModal(true)}>
+                    Audit log
+                  </DropdownMenuItem>
+                </DropdownMenuGroup>
+                <DropdownMenuSeparator />
+                <DropdownMenuGroup>
+                  <DropdownSubMenu
+                    trigger={
+                      <Flex
+                        align="center"
+                        className={isWatching ? "font-weight-bold" : ""}
+                      >
+                        <PiEye style={{ marginRight: "5px" }} size={18} />
+                        <span className="pr-5">
+                          {isWatching ? "Watching" : "Not watching"}
+                        </span>
+                      </Flex>
+                    }
                   >
-                    Edit phases
-                  </button>
-                )}
-                {canRunExperiment && growthbook.isOn("bandits") && (
-                  <ConvertBanditExperiment
-                    experiment={experiment}
-                    mutate={mutate}
-                  />
-                )}
-                <button
-                  className="dropdown-item"
-                  onClick={() => setAuditModal(true)}
-                >
-                  Audit log
-                </button>
-                <hr className="mx-4 my-2" />
-                <WatchButton
-                  itemType="experiment"
-                  item={experiment.id}
-                  className="dropdown-item text-dark"
-                />
-                <button
-                  className="dropdown-item"
-                  onClick={() => setWatchersModal(true)}
-                >
-                  View watchers{" "}
-                  <span className="badge badge-pill badge-info">
-                    {usersWatching.length}
-                  </span>
-                </button>
-                {canEditExperiment ||
-                duplicate ||
-                canRunExperiment ||
-                (hasUpdatePermissions &&
-                  experiment.archived &&
-                  permissionsUtil.canCreateReport(experiment) &&
-                  snapshot) ? (
-                  <hr className="mx-4 my-2" />
-                ) : null}
-                {canEditExperiment && (
-                  <button
-                    className="dropdown-item"
-                    onClick={() => setShareModalOpen(true)}
+                    <DropdownMenuItem
+                      onClick={async () =>
+                        await handleWatchUpdates(!isWatching)
+                      }
+                    >
+                      {isWatching ? "Stop" : "Start"} watching
+                    </DropdownMenuItem>
```

Comment:
Would love feedback on this approach. It works, but `DropdownMenuItem` doesn't formally accept an async callback function, so while it works, there isn't a great way to surface any errors that may come up. I've got it wrapped in a try/catch so errors won't cause the app the crash, and I'm logging the errors to the console, but I feel like we should improve this experience.

This also applies to the `Create shareable report` if that fails, the error is just quietly logged to the console.

We could create an error banner that lives on the `Experiment Header` that we could display if/when there is an error with one of the `MoreMenu` items. It'd need to be clearable, I guess.